webui/react/src/pages/ExperimentDetails/ExperimentInfoBox.tsx

```diff
@@ -70,7 +70,9 @@ const ExperimentInfoBox: React.FC<Props> = ({ experiment, onChange }: Props) =>
     },
     {
       content: bestValidation &&
-      `${humanReadableFloat(bestValidation)} (${config.searcher.metric})`,
+        <>
+          <HumanReadableFloat num={bestValidation} /> {`(${config.searcher.metric})`}
+        </>,
```

Comment:
**blocking:** this doesn't work either. (I wonder if events are not being passed to the component somehow)